opensrp-chw/src/main/java/org/smartregister/chw/actionhelper/ChildDevelopmentScreeningActionHelper.java

```diff
@@ -72,10 +72,10 @@ public String evaluateSubTitle() {
 
     @Override
     public BaseAncHomeVisitAction.Status evaluateStatusOnPayload() {
-        if (StringUtils.isBlank(child_development_issues)) {
```

Comment:
@devcolonel  This handled "Null point exception" onBackpress without filling the module form